diff --git a/dist/node/chunks/dep-Dq2t6Dq0.js b/dist/node/chunks/dep-Dq2t6Dq0.js
index 5e784ff4ecc7e57da946d4b98c7741e95d5d26ba..c0db9d38dbb4c0149950397171868425281a9baa 100644
--- a/dist/node/chunks/dep-Dq2t6Dq0.js
+++ b/dist/node/chunks/dep-Dq2t6Dq0.js
@@ -3000,7 +3000,7 @@ const getRequireResolver = (extensions, detectCyclesAndConditional, currentlyRes
                 return (
                   (await getTypeForImportedModule(
                     (
-                      await this.load({ id: resolved.id })
+                      await this.load(resolved)
                     ).meta.commonjs.resolved,
                     this.load
                   )) !== IS_WRAPPED_COMMONJS
